tasks/install_all_vms.yml

```diff
@@ -0,0 +1,6 @@
+- name: Push vm configuration for {{ vm.host }} @{{ vm.org }}
+  include_tasks: push_vm.yml
```

Comment:
is this inclusion really necessary? You could just merge the two files because at this stage nothing is looped.